src/sardana/taurus/core/tango/sardana/macroserver.py

```diff
@@ -94,7 +95,11 @@ def eventReceived(self, src, type, evt_value):
             self.fireEvent(None)
         elif type != TaurusEventType.Config:
             if evt_value:
-                self.fireEvent(evt_value.value)
+                if taurus_version.split(".")[0] == '4':
```

Comment:
This would fail for "Taurus 5.0.0"
Use `>=` instead (and cast to int)

... or even better, use:

```python
from distutils.version import LooseVersion
if LooseVersion(taurus_version) >= '4':
    ...
```